client/api/eventsApi.js

```diff
@@ -12,23 +19,38 @@ export function addNewEvent (id, event) {
     .catch((error) => console.log(error))
 }
 
+/**
+ * Request array of events from projects table
+ * @returns {array} All events
+ */
 export function showAllEvents () {
   return request
     .get('/api/v1/events')
     .set(acceptJsonHeader)
     .then((res) => res.body)
     .catch((error) => console.log(error))
 }
-
-export function joinEvent (id, eventID) {
+/**
+ * Add userID and EventId to users_projects table
+ * @param {string} id - User Id
+ * @param {string} eventId - eventId
```

Comment:
❔ Are these meant to be strings or numbers?
❔ Should we rename `id` to `userId`, so that its meaning is not ambiguous?

Reply:
they are numbers but you receive them as strings eg "1" and are converted to numbers when the route receives them. 
It does make it a little confusing should I expand the description to cover this?
"User ID number as a string" ? - or something like that?

Reply:
I'm on board with eventId and userId to make it clear